hw/ip/clkmgr/data/clkmgr.sv.tpl

```diff
@@ -371,15 +371,15 @@
     .q_o(${k}_sw_en)
   );
 
-  lc_tx_t ${k}_scanmode;
-  prim_lc_sync #(
+  prim_mubi_pkg::mubi4_t ${k}_scanmode;
+  prim_mubi4_sync #(
     .NumCopies(1),
     .AsyncOn(0)
   ) u_${k}_scanmode_sync  (
     .clk_i(1'b0),  //unused
     .rst_ni(1'b1), //unused
```

Comment:
it just occurred to me that even without sync flops it may make sense to feed in the clock / reset so that the assertions still work... should we change that?

Reply:
o fair point. I think i only hardwired it to 0 because i was getting a bunch of "unused clocks / resets" lint warnings.

Reply:
o wait one thing... without the clock, i think even the KNOWN / INIT checks should work.
But the other asserts on the other hand, are more or less only needed if we actually have a domain crossing right? This one basically assumes there is not, so I feel like the asserts wouldn't have much of a purpose either.

Reply:
yeah it is probably ok to omit (but it wouldn't hurt either, since the properties should be true).

as for the lint warnings, we could maybe try to add a common waiver for that (but I am not sure whether that really works because sometimes these clock/reset warnings only pop up with more context, making it difficult to scope them to within this primitive)...